core/src/main/java/org/elasticsearch/index/query/MultiMatchQueryBuilder.java

```diff
@@ -213,6 +215,7 @@ public MultiMatchQueryBuilder(StreamInput in) throws IOException {
         fuzziness = in.readOptionalWriteable(Fuzziness::new);
         prefixLength = in.readVInt();
         maxExpansions = in.readVInt();
+        fuzzyTranspositions = in.readBoolean();
```

Comment:
This break wire backwards compatibility which we can't do. Have a look at https://github.com/elastic/elasticsearch/blob/master/core/src/main/java/org/elasticsearch/action/bulk/byscroll/BulkByScrollTask.java#L192-L196 for an example of how we keep it.